pkg/types/intoto/v0.0.2/entry.go

```diff
@@ -294,7 +294,7 @@ func (v *verifier) Public() crypto.PublicKey {
 	return nil
 }
 
-func (v *verifier) Sign(data []byte) (sig []byte, err error) {
+func (v *verifier) Sign(context context.Context, data []byte) (sig []byte, err error) {
```

Comment:
nit, unused param should be `_`

Is this because the signature interface changed?

Reply:
yes.

will update

Reply:
done @haydentherapper PTAL